### PR TITLE
Add initial webidl to the spec.

### PR DESCRIPTION
**Pull request by wanderview**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/urlpattern/pull/51.html" title="Last updated on Jul 15, 2021, 3:33 PM UTC (c31284c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/urlpattern/51/46664c5...c31284c.html" title="Last updated on Jul 15, 2021, 3:33 PM UTC (c31284c)">Diff</a>

### CODE REVIEW

**Comment by wanderview** [comment]
@domenic can you review this small PR to stub in the URLPattern webidl.  I also brought over some of the whatwg styling from app-history as it seems likely URLPattern will one day end up in whatwg.

File: spec.bs [hunk 1]
```diff
@@ -18,6 +18,10 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 </pre>
 
+<pre class="link-defaults">
+spec: html; type: element; text: a
+</pre>
```
Comment (domenic) [inline spec.bs:23]: Shouldn't be necessary for you

File: spec.bs [hunk 2]
```diff
@@ -28,8 +32,111 @@ Markup Shorthands: markdown yes
...
 
 <script src="https://resources.whatwg.org/file-issue.js" async></script>
 
-TODO write some spec text! (Test)
+<h2 id="global">The {{URLPattern}} class </h2>
+
+<xmp class="idl">
+typedef (USVString or URLPatternInit) URLPatternInput;
+
+[ Exposed=(Window,Worker) ]
```
Comment (domenic) [inline spec.bs:95]: Nit: `[Exposed=(Window,Worker)]` with no spaces is more idiomatic.

**Review by domenic — APPROVED** [review]
LGTM with nits

File: spec.bs (hunk 1 above)
Comment (wanderview) [reply spec.bs]: Removed.

File: spec.bs (hunk 2 above)
Comment (wanderview) [reply spec.bs]: Done.

**Pull request merged**